lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -59,16 +59,26 @@ def on_command(node)
         return unless message_value == "test" && node.arguments.parts.any?
 
         first_argument = node.arguments.parts.first
-        return unless first_argument.is_a?(SyntaxTree::StringLiteral)
+
+        parts = case first_argument
+        when SyntaxTree::StringConcat
+          # We only support two lines of concatenation on test names
```

Comment:
I wonder if it would be worth proposing something in Syntax Tree to make these easier to work with?

Reply:
I can't see an alternative structure. How could we make the tree simpler?

Reply:
I think it could potentially be in scope, but given that we're switching to YARP soon, let's move forward with this and re-evaluate later.